app/models/organism.rb

```diff
@@ -31,4 +31,26 @@ def primer_sets_config
 
     [config, primer_sets]
   end
+
+  def lineage_variants_data(data_server, organism_slug)
```

Comment:
this is now testable... worth adding a test (maybe with copilot?)

Reply:
Agreed, I'll try Copilot to see what it can do (and write one manually if it messes up too much of course).